mod-inventory/src/main/resources/folijet/mod-inventory/features/inventoryFeatureTest.feature

```diff
@@ -12,7 +12,7 @@ Feature: inventory
       Given def instance = call read(utilsPath+'@CreateInstance') { source:'FOLIO', title:'TestInstance' }
       And def instanceId = instance.id
 
-      Given def holdings = call read(utilsPath+'@CreateHoldings') { instanceId:'#(instanceId)' }
+      Given def holdings = call read(utilsPath+'@CreateHoldings') { instanceId:'#(instanceId)', sourceId: 'dc3fa469-d5e2-4b59-85d1-8b826e3219cf' }
```

Comment:
could you say please do we really need to specify "sourceId" for the util @CreateHoldings scenario? Just would like to understand because if I'm not mistaken the sourceId value already hardcoded in the holdings template that is used by util @CreateHoldings scenario

Reply:
@RuslanLavrov  Good point! Let me double-check it.

Reply:
@RuslanLavrov Yes, you are right! Thank you! I pushed the latest changes.

Reply:
understood, thank you @VRohach!